decidim-core/spec/lib/events/email_event_spec.rb

```diff
@@ -39,7 +39,7 @@ class TestEvent < Decidim::Events::BaseEvent
 
       describe ".has_button?" do
         it "responds and returns false" do
-          expect(subject).not_to have_button
+          expect(subject).not_to have_button # rubocop:disable Capybara/NegationMatcher
```

Comment:
Is this a false positive from rubocop-capybara? Like it's kicking this rule when it shouldn't as the class that we're testing is a concern?

Reply:
Actually, this is a clash of names. The spec is actually testing that the boolean method "has_button?" is false. And the cell itself does not respond to has_no_button?
The actual pipeline fail is: 

```
 2nd Try error in ./spec/lib/events/email_event_spec.rb:41:
expected #<Decidim::TestEvent:0x00007f306ebe0a00 @event_name="test", @resource=#<Decidim::User id: 429, email:...fficialized_at: nil, officialized_as: nil, admin_terms_accepted_at: nil>, @user_role=nil, @extra={}> to respond to `has_no_button?`
```

Reply:
Ah OK, I thought that the negation matcher worked in another way (like `!subject.has_button?` instead of `subject.has_no_button?`)